src/HeaderCell.tsx

```diff
@@ -50,7 +50,15 @@ export default function HeaderCell<R, K extends keyof R>({
     return createElement(renderer, { column, allRowsSelected, onAllRowsSelectionChange });
   }
 
-  let cell = getCell();
+  const DraggableHeaderCell = props.draggableHeaderCell;
+  let cell = column.draggable && DraggableHeaderCell ? (
+    <DraggableHeaderCell
+      column={column}
+      onHeaderDrop={props.onHeaderDrop!}
+    >
+      {getCell()}
+    </DraggableHeaderCell>
+  ) : getCell();
```

Comment:
Why are we moving the draggable header wrapping here?

Reply:
It's the preview thing I mentioned before. It's a bug only in Chrome. Without this change, the dragging preview will show the whole column headers rather than the dragged one.